sql/expression/function/queryinfo.go

```diff
@@ -1,6 +1,10 @@
 package function
 
-import "github.com/dolthub/go-mysql-server/sql"
+import (
+	"fmt"
+	"github.com/dolthub/go-mysql-server/sql"
```

Comment:
Adding a newline to keep the formatting checker happy.

```suggestion

	"github.com/dolthub/go-mysql-server/sql"
```